app/jobs/regular/answer_similar_questions.rb

```diff
@@ -21,7 +21,11 @@ def execute(args)
         # send a PM to user to recommend similar topics to answer
         if suggested_topics.topics.length > 1 &&
              !!user.custom_fields[SolvedReminders::USER_CUSTOM_FIELD_NAME] == false
-          raw = I18n.t("answer_similar_questions.message").dup
+          raw =
+            I18n.t(
+              "answer_similar_questions.message",
+              solved_post_url: "#{Discourse.base_url}#{solved_post.url}",
+            ).dup
```

Comment:
Does it have to be duplicated? 😃

Reply:
I don't see why `dup` is required. Removed in latest commit.